enterprise/server/auth/auth.go

```diff
@@ -113,7 +115,17 @@ func (c *Claims) IsAdmin() bool {
 	return false
 }
 
-func assembleJWT(ctx context.Context, userID, groupID string, allowedGroups []string) (string, error) {
+func (c *Claims) HasCapability(cap akpb.ApiKey_Capability) bool {
+	return int32(cap)&c.Capabilities > 0 || c.IsAdmin()
```

Comment:
Wasn't sure whether this `c.IsAdmin()` check makes sense or not?

Reply:
I would just leave admin out of it since we don't know if having a capability is something we'd want or not.

Reply:
OK, removed